about.pug

```diff
@@ -8,6 +8,17 @@ block prepend title
 
 block content
     h1 About
+    p.
+        Puyo Puyo VS is a client that can connect to internet lobbies in order to play Puyo Puyo Matches against up to 10 people at the same time.
+        Features the ability to customize assets used ingame, modes such as Tsu, which is the standard Puyo ruleset, as well as Fever, Non-stop Fever, and more, and
+        chatrooms to organize matches.
+    p
+        br
+    p.
+        Original and thankfully outdated description below, Puyo has multiple official Western releases now, but Puyo Puyo VS remains as a free,
```

Comment:
I think we can just remove the original description and not need this note at all. However, it would be a good idea to add a line somewhere that describes the game as having been originally developed by Hernan, just to ensure it's written on here somewhere. It can probably fit anywhere into the first paragraph.